decentraland/components/presentational/LinkToEtherscan.js

```diff
@@ -1,34 +1,44 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Button, Icon } from "native-base";
-import { showError, openURL } from "@helpers";
+import {
+  showError,
+  openURL,
+  getNetworkName,
+  getTransactionHashFromAction,
+} from "@helpers";
 
-const _openEtherscanOf = async address => {
-  const url = `https://etherscan.io/address/${address}`;
+const supportedNetworks = ["ropsten"];
+
+const _openEtherscanOf = async action => {
+  const networkName = getNetworkName();
+  const transactionHash = await getTransactionHashFromAction(action);
+  const url = `https://${networkName}.etherscan.io/tx/${transactionHash}`;
```

Comment:
We could make the function even more abstract and build the entire Etherscan transaction URI using a helper. Not sure if that's a better or worse option, though. It does keep the mobile dev from having to see the word tx hash, which is a win.

Reply:
Done.
Make sense rename the component `LinkToEtherscan` to another name more generic as `LinkToBlockchain` or `LinkToReceipt` since we could in future use another blockchain explorer.

Reply:
Yes, I really like that idea!

Reply:
Which one do you like more?

Reply:
Doesn't matter too much between the two. Let's go with `LinkToBlockchain` though.

Reply:
Done.

Reply:
Thanks! Resolving this